docs/recipes/ec2-eks-metrics-go-adot-ampamg.md

```diff
@@ -1,12 +1,314 @@
+      },
+      "Action": "sts:AssumeRoleWithWebIdentity",
+      "Condition": {
+        "StringEquals": {
+          "oidc.eks.aws_region.amazonaws.com/id/openid:sub": "system:serviceaccount:adot-col:amp-iamproxy-ingest-service-account"
+        }
+      }
+    }
+  ]
+}
+```
+6. Choose the Permissions tab and make sure that the following permissions policy is attached to the role. 
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "aps:RemoteWrite",
+                "aps:GetSeries",
+                "aps:GetLabels",
+                "aps:GetMetricMetadata"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+[Reference: Set up metrics ingestion using AWS Distro for Open Telemetry](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-onboard-ingest-metrics-OpenTelemetry.html)
+
+### Setup ADOT Collector 
+
+#### Download your template file
+
+Depending on if your EKS cluster uses AWS Fargate or not we will use a different template file for the rest of this guide.
+
+The following template file uses a deamonset for it's configuration [prometheus-daemonset.yaml](ec2-eks-metrics-go-adot-ampamg/prometheus-daemonset.yaml).
+
+AWS Fargate does not support deamonsets so you can use the following template [prometheus-fargate.yaml](ec2-eks-metrics-go-adot-ampamg/prometheus-fargate.yaml), if your cluster was setup with AWS Fargate.
+
+
+#### Edit your template file
+
+In this example, the ADOT Collector configuration uses an annotation (scrape=true) to tell which target endpoints to scrape. This allows the ADOT Collector to distinguish the sample app endpoint from kube-system endpoints in your cluster. You can remove this from the re-label configurations if you want to scrape a different sample app. 
+
+Use the following steps to edit the downloaded file for your environment:
+
+1\. Replace **<REGION\>** with your current Region. 
+
+2\. Replace **<YOUR_ENDPOINT\>**  with your AMP workspace endpoint URL.
+
+Get your AMP endpoint url by executing the following query:
+```
+$aws amp describe-workspace \ 
+    --workspace-id `aws amp list-workspaces --alias prometheus-sample-app --query 'workspaces[0].workspaceId' --output text` \
+    --query 'workspace.prometheusEndpoint'
+```
+
+3\. Finally replace your **<YOUR_ACCOUNT_ID\>**  with your current account ID.
+
+The following command will return the account ID for the current session:
+```
+aws sts get-caller-identity --query Account --output text
+```
+<br/>
+
+#### Deploy the template to your cluster
+
+```
+$ kubectl apply -f prometheus-fargate.yaml
+```
+
+In case you used the deamonset template use the following command
+```
+$ kubectl apply -f prometheus-daemonset.yaml
+```
+
+You can verify that the ADOT Collector has started with this command:
+
+```
+$ kubectl get pods -n adot-col
```

Comment:
The namespace issue needs to be fixed if using fargate profile.

Reply:
This namespace should now be available by adding it to the cluster-config.yaml